psyneulink/core/components/functions/nonstateful/objectivefunctions.py

```diff
@@ -205,7 +185,11 @@ class Parameters(ObjectiveFunction.Parameters):
         metric = Parameter(ENERGY, stateful=False)
         metric_fct = Parameter(None, stateful=False, loggable=False)
         transfer_fct = Parameter(None, stateful=False, loggable=False)
-        normalize = Parameter(False, stateful=False)
+        normalize = FunctionParameter(
+            False,
+            function_name='metric_fct',
+            function_parameter_name=NORMALIZE,
```

Comment:
Specifying function_parameter_name isn't needed if it's the same as the parameter name, but it doesn't bother me if this is more readable.

Reply:
removed